### PR TITLE
Fixed category buttons

### PR DESCRIPTION
**Pull request by mirjoy**

### CODE REVIEW

File: app/views/menu/index.html.erb [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
   <% @items.each do |item| %>
-    <div class="col-sm-4 items">
+    <div class="col-sm-3 items">
       <%= item.name %>
-      <%= button_to "Add to Cart", cart_items_path(item: item) %>
+      <%= button_to "Add to Cart", cart_items_path(item: item), class: "btn btn-default" %>
+      <div class="btn-group btn-group-xs" role="group">
       <% item.categories.each do |item_category| %>
-      <p><%= link_to item_category.display_name, menu_category_path(item_category) %></p>
+        <%= link_to item_category.display_name, menu_category_path(item_category), class: "btn btn-default" %>
```
Comment (indiesquidge) [inline app/views/menu/index.html.erb]: Can you refer to these as just `item_category` instead of `menu_category_path(item_category)`?

**Comment by indiesquidge** [comment]
Too many ambiguous switches between the term "Log In"/"Login"/"Sign In"/"Signin"/etc. We need to pick one and stick with it.

**Comment by teresafinn** [comment]
Styling looks great, nicely done!

**Comment by indiesquidge** [comment]
Shouldn't have merged this just yet since it will now cause conflicts with the changes from "log in" to "sign in". That's okay though, we'll figure it out.

**Pull request merged**